lib/datastore/request.js

```diff
@@ -224,6 +226,21 @@ DatastoreRequest.prototype.get = function(keys, callback) {
  *     }
  *   }
  * ], function(err) {});
+ *
+ * //-
+ * // Explicitly attempt to 'insert' a specific entity.
+ * //-
+ * var userKey = dataset.key(['User', 'chilts']);
+ *
+ * dataset.save([
+ *   {
+ *     key: userKey,
+ *     method: 'insert', // force the method to 'insert'
+ *     data: {
+ *       fullName: 'Andrew Chilton'
+ *     }
```

Comment:
No way, you're part of this library forever now. https://github.com/GoogleCloudPlatform/gcloud-node/commit/d3fe788b6a4c59269d17dad246f108d8160a7b3b

Seriously though, I just used your name there to be funny. If you want me to remove the honorable mentions now, I'll be happy to. :+1: